packages/compatibility/src/utils/markdown.ts

```diff
@@ -91,7 +91,7 @@ const singleImplementationTemplate = `
 `;
 
 const apolloIcon: string =
-  '<img style="display:inline-block; height:1em; width:auto;" alt="Maintained by Apollo" src="https://apollo-server-landing-page.cdn.apollographql.com/_latest/assets/favicon.png"/>';
+  '<img style="display:inline-block; height:1em; width:auto;" alt="Maintained by Apollo" src="https://raw.githubusercontent.com/apollographql/apollo-federation-subgraph-compatibility/96e13acebd8d699127f2bae142622fbd4c392ecf/docs/assets/horizon_logo.png"/>';
```

Comment:
our [docs](https://www.apollographql.com/docs/federation/building-supergraphs/supported-subgraphs/) will be pointing to this github repo for the icon -> is this fine? or do we have a better place to host it?

Reply:
Double checking with @apollographql/dx-engineering

Reply:
@dariuszkuc , they're good with it :)